app/uk/gov/hmrc/helptosavefrontend/controllers/HelpToSaveAuth.scala

```diff
@@ -36,34 +36,31 @@ class HelpToSaveAuth(app: Application) extends FrontendController with Authorise
 
   override def env: Environment = Environment(app.path, app.classloader, app.mode)
 
+  private type HtsAction = Request[AnyContent] ⇒ HtsContext ⇒ Future[Result]
 
-  private type HtsAction = Request[AnyContent] => Future[Result]
-  private type HtsActionWithEnrolments = Request[AnyContent] => Option[NINO] => Future[Result]
-
-  def authorisedForHtsWithEnrolments(action: HtsActionWithEnrolments): Action[AnyContent] = {
+  def authorisedForHtsWithEnrolments(action: HtsAction): Action[AnyContent] =
```

Comment:
this could almost be renamed to `authorisedForHtsWithNino` since the only difference with `authorisedForHts` is that it returns a NINO if possible